nav2_waypoint_follower/CMakeLists.txt

```diff
@@ -21,6 +26,7 @@ find_package(nav2_util REQUIRED)
 find_package(tf2_ros REQUIRED)
 find_package(nav2_core REQUIRED)
 find_package(pluginlib REQUIRED)
+find_package(robot_localization REQUIRED)
```

Comment:
Please add robot_localization to the `tools/underlays.repos` file. That lets us add RL in the underlay workspace if its a requirement to build. In CI we use ROS2 master, so anything not included in the ROS2 master builds needs to be added so we can build it against ROS2 master for building nav2. That's why CI is failing right now.

Reply:
I think after https://github.com/ros-planning/navigation2/pull/2111/commits/c2f4d95d6f53df101c562ad0adbb1e91638a397d
CI is OK, at least in terms of required dependency packages.